dbms/src/Dictionaries/FlatDictionary.cpp

```diff
@@ -487,6 +494,7 @@ void FlatDictionary::setAttributeValue(Attribute & attribute, const Key id, cons
         case AttributeUnderlyingType::UInt16: setAttributeValueImpl<UInt16>(attribute, id, value.get<UInt64>()); break;
         case AttributeUnderlyingType::UInt32: setAttributeValueImpl<UInt32>(attribute, id, value.get<UInt64>()); break;
         case AttributeUnderlyingType::UInt64: setAttributeValueImpl<UInt64>(attribute, id, value.get<UInt64>()); break;
+        case AttributeUnderlyingType::UInt128: setAttributeValueImpl<UInt128>(attribute, id, value.get<UInt128>()); break;
```

Comment:
I have doubts about this. In all other cases, a 64-bit value is used, but in my case 128-bit value.

Reply:
This is Ok.

In `Field` we have just UInt64, Int64 to store anything of 8, 16, 32, 64bit numbers.
And also UInt128 as separate variant.